src/Microsoft.Identity.Web.DownstreamApi/DownstreamApi.cs

```diff
@@ -219,7 +302,7 @@ public Task<HttpResponseMessage> CallApiForAppAsync(
             return clonedOptions;
         }
 
-        internal static HttpContent? SerializeInput<TInput>(TInput input, DownstreamApiOptions effectiveOptions)
+        internal static HttpContent? SerializeInput<TInput>(TInput input, DownstreamApiOptions effectiveOptions, JsonTypeInfo<TInput>? inputJsonTypeInfo = null)
```

Comment:
Adding an optional parameter is a binary breaking change. I think this should be an additional overload and add it to NET8 only section.

Reply:
@pmaytak Thanks, I've updated it to have an overload and keep the original signature. I've put the implementation into a different method though - is this enough re binary breaking change for an internal method? Or does the original implementation need to stay the same?

Reply:
I think this is good.